mscore/inspector/inspector.cpp

```diff
@@ -1216,9 +1216,11 @@ InspectorCaesura::InspectorCaesura(QWidget* parent) : InspectorElementBase(paren
             c.title->setText(b->accessibleInfo());
 
       const std::vector<InspectorItem> il = {
-            { Pid::PAUSE,  0, c.pause,         c.resetPause         }
-            };
+            { Pid::PAUSE    ,  0, c.pause,         c.resetPause         },
+            { Pid::PLACEMENT,  0, c.placement,     c.resetPlacement     }
+      };
```

Comment:
Indent, banner style

Reply:
The existing pause did some fancy indent. I just adapted it to the new placement line.

Reply:
You should have just added the placement line. You should not have removed the indentation from the closing brace.

Reply:
Right, will change it back.